catalyst-gateway/bin/src/event_db/config.rs

```diff
@@ -40,7 +43,8 @@ impl EventDB {
 
         let mut follower_configs = Vec::new();
         for row in rows {
-            let network = row.try_get("id3")?;
```

Comment:
Can we refactor the magic string `id3` to something more concrete and self-explanatory?

Reply:
What if we define a new type, like this:

```rust
enum CardanoNetwork {
    Id,
    Id2,
    Id3,
}

impl CardanoNetwork {
    fn as_str(&self) -> &'static str {
        match self {
            CardanoNetwork::Id => "id",
            CardanoNetwork::Id2 => "id2",
            CardanoNetwork::Id3 => "id3",
        }
    }
}


fn main() {
   let cardanoNetwork = CardanoNetwork::Id;
    println!("The CardanoNetwork is: {}", cardanoNetwork.as_str());
}
```

Reply:
Yea, thank you, totally make sense